documentation/sphinx/source/release-notes/release-notes-730.rst

```diff
@@ -2,6 +2,16 @@
 Release Notes
 #############
 
+7.3.27
+======
+* Same as 7.3.26 release with AVX disabled.
+
+7.3.26
+======
+* Fixed exclude command in fdbcli, allows excluding servers by locality.
+* Updated RocskDB version to 8.6.7. `(PR #11043) <https://github.com/apple/foundationdb/pull/11043>`_
+* Various bug fixes in tests
```

Comment:
What kind of fixes? if it's simulation test fixes, we can omit them in the release notes.

Reply:
Just browse the changes. It's worth mentioning the following PRs:

- add fdb_c_apiversion.g.h to osx package 
- https://github.com/apple/foundationdb/pull/11039
- https://github.com/apple/foundationdb/pull/11016

Reply:
- https://github.com/apple/foundationdb/pull/10998